src/client/components/Analytics/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import { Table } from '@instructure/ui-table';
+import axios from 'axios';
+import { ltikPromise } from '../../services/ltik';
+
+const Analytics = () => {
+  const [analytics, setAnalytics] = useState([]);
+
+  const getAnalytics = () => {
+    ltikPromise.then(ltik => {
+      axios.get(`/api/getanalytics?ltik=${ltik}`).then(res => {
+        setAnalytics(res.data);
+      });
+    });
+  };
+
+  useEffect(getAnalytics, []);
+  console.log(analytics);
+  const analyticsBody = analytics
+    ? analytics.map(stat => (
+        <Table.Row>
+          <Table.RowHeader>{stat.contextId}</Table.RowHeader>
+          <Table.Cell>{stat.shortname}</Table.Cell>
+          <Table.Cell>{stat.numMembers}</Table.Cell>
+          <Table.Cell>{stat.total_reserve_clicks}</Table.Cell>
```

Comment:
I think it's a little odd to have both snake case and camel case for these properties

Reply:
Good point. I'll change that.

Reply:
@kilometerskang @finnzink If we want to fix this please create a ticket for code cleanup and to add this rule to eslint to help find all the places this naming convention needs to be fixed: https://eslint.org/docs/rules/camelcase